### ISSUE TITLE
Error with jquery-rails when an app is created with the -J flag

### ISSUE THREAD

**Issue by em0ral** [fortuity/rails3-mongoid-devise#18]
gem 'jquery-rails', '>= 0.2.7'

adds config.action_view.javascript_expansions[:defaults] = %w()

this empty array causes error and should be changed to 

config.action_view.javascript_expansions[:defaults] = %w(jquery rails)

**Comment by fortuity** [comment]
Were you following the instructions in the tutorial for using jquery-rails to automatically install jQuery?

Perhaps I should eliminate the option of using jquery-rails. I believe the instructions for manual installation are accurate. Can you confirm that?

-- Daniel

**Comment by em0ral** [comment]
Just went thru your tutorial once again and I'm positive that jquery-rails gem does not add (jquery rails). Has to be added manually

**Comment by fortuity** [comment]
Thanks for reporting the problem. I've investigated and submitted this issue

https://github.com/indirect/jquery-rails/issues#issue/54

for the jquery-rails gem.

If an app is created with 

`rails new testapp -J` 

(using the "-J" flag to eliminate the Prototype files)
then in the _config/application.rb_ file
`config.action_view.javascript_expansions[:defaults] = %w()`
is not commented out and _lib/jquery-rails.rb_ cannot override
`config.action_view.javascript_expansions[:defaults]` to include the jquery files and you'll get an error "No expansion found for :defaults" when you run your app.

In conclusion: Don't use the `-J` flag to generate an app if you are going to use the jquery-rails gem. If you use the `-J` flag, then install jQuery manually.

I've updated the tutorial to explain this.

-- Daniel

**Comment by em0ral** [comment]
Not a problem...Im just learning rails so trying to be active...
